client/users.go

```diff
@@ -47,22 +47,41 @@ type CreateUserOptions struct {
 	ForceManaged bool   `json:"force-managed,omitempty"`
 }
 
+// RemoveUserOptions holds options for removing a local system user.
+//
+// Username indicates which user to remove.
+type RemoveUserOptions struct {
+	Username string `json:"username,omitempty"`
+}
+
+type userAction struct {
+	Action string `json:"action"`
+	*CreateUserOptions
+	*RemoveUserOptions
+}
+
+func (client *Client) doUserAction(act *userAction) ([]*CreateUserResult, error) {
```

Comment:
Since it does both add/remove now, maybe we should rename `CreateUserResult` to `UserActionResult` ?

Reply:
this is an internal helper; the public-facing API only returns this type for CreateUser; `client` doesn't expose "user action" to its users.